patches/common/chromium/chrome_process_finder.patch

```diff
@@ -0,0 +1,36 @@
+From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
+From: Milan Burda <miburda@microsoft.com>
+Date: Fri, 26 Oct 2018 20:45:49 +0200
+Subject: chrome_process_finder.patch
+
+Pass original command line
```

Comment:
Can you please add more information here what this patch is supposed to do?, and what for?

Reply:
Agreed, I believe the reason was that command line will be accessible by users when trying to launch the instance and we want the original one instead of modified args by chromium. /cc @zcbenz for more context. https://github.com/electron/electron/pull/3175

Reply:
done